src/gui/gatewayer_mock_test.go

```diff
@@ -998,3 +999,51 @@ func (m *GatewayerMock) UpdateWalletLabel(p0 string, p1 string) error {
 	return r0
 
 }
+
+// GetDefaultStatus mocked method
+func (m *GatewayerMock) GetDefaultStatus() *daemon.ConnectionsHealth {
```

Comment:
The GatewayerMock methods don't need an implementation, it is generated with `goautomock` tool (you could implement it by hand, following the example of other mocked methods in this file)

Reply:
I applied most of the changes requested in this review , but I did not understand this one in particular . What shall be done ?

Reply:
done !